.terraform/jenkins-agents-images/main.tf

```diff
@@ -0,0 +1,43 @@
+variable "terraform-images" {
+  type = map
+  default = {
```

Comment:
I would prefer to use the numbering scheme, i.e ubuntu1604/1804 as opposed to xenial/bionic